site/install.md

```diff
@@ -84,7 +85,7 @@ and GLib (C) package with:
 brew install apache-arrow-glib
 ```
 
-### C++ and GLib (C) Packages for Debian GNU/Linux, Ubuntu and CentOS (Unofficial)
+### C++ and GLib (C) Packages for Debian GNU/Linux, Ubuntu and CentOS
```

Comment:
They are "unofficial" just for 0.13.0 by packaging mistake.
https://lists.apache.org/thread.html/43ecf510ea3f04a375854f70886195c140d0e96854b43286e389e3c0@%3Cdev.arrow.apache.org%3E
Normally, this is "official" (voted) binary packages.

Reply:
Ok, so we can move this above the "unofficial" marker when we do 0.14.0.

Reply:
https://issues.apache.org/jira/browse/ARROW-5444 for that fix later.